src/common/i18n.js

```diff
@@ -0,0 +1,164 @@
+const defaultLanguage = 'en';
+
+const i18n = {
```

Comment:
Could you move this const to separate file please? Look for other `constants.js` files

Reply:
Done.